arbitrum-docs/build-decentralized-apps/how-to-use-timeboost.mdx

```diff
@@ -0,0 +1,473 @@
+---
+title: 'How to use Timeboost'
+description: Learn how to use timeboost
+author: jose-franco
+content_type: how-to
+---
+
+Timeboost is a new transaction ordering policy for Arbitrum chains. With Timeboost, anyone can bid for the right to access an express lane on the sequencer for faster transaction inclusion.
+
+In this how-to, you'll learn how to bid for the right to use the express lane, how to submit transactions through the express lane, and how to transfer that express lane rights to someone else. To learn more about Timeboost, refer to the [gentle introduction](/how-arbitrum-works/timeboost/gentle-introduction.mdx).
+
+This how-to assumes that you're familiar with:
+
+- [How Timeboost works](/how-arbitrum-works/timeboost/gentle-introduction.mdx)
+- [viem](https://viem.sh/), since the snippets of code present in the how-to use this library
+
+## How to submit bids for the right to be the express lane controller
+
+To use the express lane for faster transaction inclusion, you must win an auction for the right to be the express lane controller for a specific round.
+
+::::info
+
+Remember that, by default, each round lasts 60 seconds, and the auction for a specific round closes 15 seconds before the round starts. These default values can be configured on a chain using the `roundDurationSeconds` and `auctionClosingSeconds` parameters, respectively.
+
+::::
+
+Auctions are held in an auction contract, and bids are submitted to an autonomous auctioneer, that also communicates with the contract. Let's take a look at the process of submitting bids and finding out the winner of an auction.
```

Comment:
```suggestion
Auctions are in an auction contract, and bids get submitted to an autonomous auctioneer who communicates with the contract. Let's look at the process of submitting bids and finding out the winner of an auction.
```

Reply:
I kept `held` here. It seems to me that the sentence is not completed by just saying "Auctions are in an auction contract".